src/hooks/useReadingProblems.ts

```diff
@@ -27,7 +31,15 @@ export default function useReadingProblems({ navigation, route }: Props) {
 
   const onTimeComplete = useCallback(
     (skipped: boolean) => {
-      const statistics = { passagesRead, completed: !skipped }; // eslint-disable-line
+      const averageTime =
+        passagesRead === 0 || skipped ? 0 : TOTAL_TIME / passagesRead;
+      const statistics = {
+        passagesRead,
+        completed: !skipped,
+        wordsPerMinute: wordsRead / TOTAL_TIME,
+        timePerPassage: averageTime,
+      }; // eslint-disable-line
```

Comment:
If a person skips the section, `timePerPassage` and `wordsPerMinute` metrics should only account for the time the patient spent on this section (not TOTAL_TIME). If a patient spends 30 seconds in the reading section and skips, we will get skewed results if we use TOTAL_TIME instead of 30 seconds, which makes it incomparable. Moreso, `timePerPassage` should not be 0 if the patient skips the section. It should be 0 only if no passages were read.